docker-compose.yml

```
@@ -19,4 +19,4 @@ services:
       - MQTT_PASSWORD
 
       # Selected robot package
-      - ROBOT_PACKAGE=isar_robot
+      - ISAR_ROBOT_PACKAGE=isar_turtlebot
```

```suggestion
      - ISAR_ROBOT_PACKAGE=isar_robot
```
Default should be isar_robot